msg/MotionPlanRequest.msg

```diff
@@ -51,6 +51,21 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
+# New joint limits to use when running time-parameterization and smoothing.
+# Note, this does not replace the values in the robot model.
+# Only applicable for TOTG and Ruckig
+# Set the 'has_limits' field to true for velocity, acceleration, and/or jerk
+# in order for that limit to be applied. Position limits are not supported.
+JointLimits[] joint_limits
+
+# If true, use the joint limits defined in the 'joint_limits' field.
+# Otherwise, use the max scaling factors defined for velocity and acceleration
+bool use_joint_limits
```

Comment:
I think this flag is redundant. Either the limits are passed and should be applied, or the default limits should be used instead. Also, the scaling factors should always be applied, even with custom joint limits

Reply:
It also seems redundant to me unless you have a specific use case in mind @mechwiz

Reply:
I did this since the computeTimeStamps functions within [time_parameterization.h ](https://github.com/mechwiz/moveit2/blob/main/moveit_core/trajectory_processing/include/moveit/trajectory_processing/time_parameterization.h#L15) only support scaling or joint limits but not both combined.
I am happy to create another overloaded function though that accepts both as arguments as I agree with your feedback.

Reply:
Note, I created this PR here https://github.com/ros-planning/moveit2/pull/1780

Reply:
done in latest commit